src/tokens.js

```diff
@@ -0,0 +1,7 @@
+export default function Tokens(spec) {
```

Comment:
Why did you re-write this (without tests) instead of using the existing code in configure?!

- fails if `token` is provided in lower case (we already have uses doing this)
- `tokenabcdef` is a valid api key, `token abcdef` is not; you can see this knowledge already baked in to `src/configure.js`.